components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java

```diff
@@ -2546,4 +2551,22 @@ private static UserStoreManager getUserStoreManager(int tenantId, String userSto
         }
         return userStoreManager;
     }
+
+    /**
+     * Check whether the authentication flow should continue upon facing a claim handling error.
+     *
+     * @return true/false Continue or break flow when facing claim handling errors.
+     */
+    public static boolean isContinueOnClaimHandlingErrorAllowed() {
+
+        String continueOnClaimHandlingErrorValue = IdentityUtil.getProperty(CONTINUE_ON_CLAIM_HANDLING_ERROR);
+
+        // If config is empty or not a boolean value, the property must be set to the default value which is true.
+        if (StringUtils.isBlank(continueOnClaimHandlingErrorValue) ||
+                !(TRUE.equalsIgnoreCase(continueOnClaimHandlingErrorValue) ||
+                        FALSE.equalsIgnoreCase(continueOnClaimHandlingErrorValue))) {
+            continueOnClaimHandlingErrorValue = CONTINUE_ON_CLAIM_HANDLING_ERROR_DEFAULT;
+        }
+        return Boolean.parseBoolean(continueOnClaimHandlingErrorValue);
+    }
```

Comment:
```
return !Boolean.FALSE.toString().equalsIgnoreCase(continueOnClaimHandlingErrorValue)
```

Reply:
Shall we use the above? would it capture all the cases?

Reply:
This works. Addressed it with a new commit.